src/lib/components/OpenWebSearchResults.svelte

```diff
@@ -11,31 +8,69 @@
 	export let classNames = "";
 	export let webSearchMessages: WebSearchUpdate[] = [];
 
-	let detailsOpen: boolean;
-	let error: boolean;
-	$: error = webSearchMessages[webSearchMessages.length - 1]?.messageType === "error";
+	$: sources = webSearchMessages.find((m) => m.sources)?.sources;
+	$: error = webSearchMessages.find((m) => m.messageType === "error");
 </script>
 
 <details
 	class="flex w-fit rounded-xl border border-gray-200 bg-white shadow-sm dark:border-gray-800 dark:bg-gray-900 {classNames} max-w-full"
-	bind:open={detailsOpen}
 >
-	<summary
-		class="align-center flex cursor-pointer select-none list-none py-1 pl-2.5 pr-2 align-text-top transition-all"
-	>
-		{#if error}
-			<CarbonError class="my-auto text-red-700 dark:text-red-500" />
-		{:else if loading}
-			<EosIconsLoading class="my-auto text-gray-500" />
-		{:else}
-			<CarbonCheckmark class="my-auto text-gray-500" />
-		{/if}
-		<span class="px-2 font-medium" class:text-red-700={error} class:dark:text-red-500={error}>
-			Web search
-		</span>
-		<div class="my-auto transition-all" class:rotate-90={detailsOpen}>
-			<CarbonCaretRight />
+	<summary class="grid min-w-72 select-none grid-cols-[40px,1fr] items-center gap-2.5 p-2">
+		<div
+			class="relative grid aspect-square place-content-center overflow-hidden rounded-lg bg-gray-100 dark:bg-gray-800"
+		>
+			<svg
+				class="absolute inset-0 text-gray-300 transition-opacity dark:text-gray-700 {!sources &&
+				!error
+					? 'opacity-100'
+					: 'opacity-0'}"
```

Comment:
```suggestion
				class="absolute inset-0 text-gray-300 transition-opacity dark:text-gray-700 opacity-0"
				class:opacity-100={!sources && !error}
```

haven't tested but I think it should work

Reply:
the element would have `opacity-0` and `opacity-100` set at the same time in classes, not ideal.